setup.py

```diff
@@ -49,10 +49,6 @@
         'sphinx==3.0.4',
         'sphinx_rtd_theme',
     ],
-    'travis': [
-        '-r stylecheck',
-        '-r docs',
-    ],
```

Comment:
It's used in `.github/workflows/pretest.yml`.
https://github.com/cupy/cupy/blob/436b09b1ed5255fae95a4e4f41d2875fa4f7fceb/.github/workflows/pretest.yml#L23

```diff
-        READTHEDOCS=True pip install -U -e .[travis]
+        READTHEDOCS=True pip install -U -e .[stylecheck,docs]
```

Reply:
🙇‍♂️